packages/contract-deposit/README.md

```diff
@@ -0,0 +1,34 @@
+# Contract deposit Tutorial
+
+`Contract deposit` is a simple demo of Arbitrum's L1-to-L2 deposit sending from a l1 contract
```

Comment:
```suggestion
`Contract deposit` is a simple demo of Arbitrum's L1-to-L2 deposit sending from a l1 contract
```
contract-deposit is a simple demonstration of how Arbitrum facilitates sending deposits from an L1 contract to L2